src/DynamoCoreWpf/ViewModels/RunSettingsViewModel.cs

```diff
@@ -266,6 +267,8 @@ public RunSettingsViewModel(RunSettings settings, HomeWorkspaceViewModel workspa
             Model.PropertyChanged += Model_PropertyChanged;
 
             this.workspaceViewModel = workspaceViewModel;
+            workspaceViewModel.Model.PropertyChanged += HomeWorkspaceModel_PropertyChanged;
```

Comment:
where can you unsubscribe from this handler?

Reply:
There is a Dispose method. I unsubscribed there